Rakefile.rb

```diff
@@ -11,7 +11,7 @@
   end
 
   desc "Remove Docker container and image"
-  task :remove_container_and_image do
+  task :stop_remove do
```

Comment:
I think, If you're wrapping multiple commands, it's logical to call the last one executed for simplicity's sake